### PR TITLE
Help section

### PR DESCRIPTION
**Pull request by pusztaienike**

### CODE REVIEW

**Review by herflis — APPROVED** [review]
ℹ

**Review by huszerlbence — APPROVED** [review]
just perfect, thank you.

File: apps/sensenet/src/components/appbar/desktop-nav-menu.tsx [hunk 1]
```diff
@@ -65,44 +92,66 @@ export const DesktopNavMenu: React.FunctionComponent = () => {
...
   }
 
-  const handleClose = (event: React.MouseEvent<EventTarget>) => {
-    if (anchorRef.current && anchorRef.current.contains(event.target as HTMLElement)) {
+  const handleClose = (
+    event: React.MouseEvent<EventTarget>,
+    ref: React.RefObject<HTMLButtonElement>,
+    setter: React.Dispatch<React.SetStateAction<boolean>>,
+  ) => {
+    if (ref.current && ref.current.contains(event.target as HTMLElement)) {
```
Comment (taki9) [inline apps/sensenet/src/components/appbar/desktop-nav-menu.tsx:113]: ref.current?.

File: apps/sensenet/src/components/appbar/desktop-nav-menu.tsx [hunk 2]
```diff
@@ -65,44 +92,66 @@ export const DesktopNavMenu: React.FunctionComponent = () => {
...
   const theme = useTheme()
   const service = injector.getInstance(PersonalSettings)
-  const { openDialog } = useDialog()
-  const [open, setOpen] = useState(false)
-  const anchorRef = useRef<HTMLButtonElement>(null)
   const currentUser = useCurrentUser()
   const repo = useRepository()
   const localization = useLocalization()
+  const { openDialog } = useDialog()
+  const [openLogoutMenu, setOpenLogoutMenu] = useState(false)
```
Comment (taki9) [inline apps/sensenet/src/components/appbar/desktop-nav-menu.tsx:99]: It is not about logout, it is about the user, so userMenu would be a much better name.
(replace logoutMenu in localization too)

File: apps/sensenet/src/components/appbar/desktop-nav-menu.tsx [hunk 3]
```diff
@@ -65,44 +92,66 @@ export const DesktopNavMenu: React.FunctionComponent = () => {
...
+      setOpenLogoutMenu(false)
+      setOpenHelpMenu(false)
     }
   }
 
-  const prevOpen = useRef(open)
+  const prevLogoutOpen = useRef(openLogoutMenu)
+  const prevHelpOpen = useRef(openHelpMenu)
+
+  useEffect(() => {
```
Comment (taki9) [inline apps/sensenet/src/components/appbar/desktop-nav-menu.tsx:131]: why is this magic useful?

File: apps/sensenet/src/components/appbar/desktop-nav-menu.tsx (hunk 1 above)
Comment (taki9) [reply apps/sensenet/src/components/appbar/desktop-nav-menu.tsx:113]: what does it do?

File: apps/sensenet/src/localization/hungarian.ts [hunk 4]
```diff
@@ -180,6 +184,17 @@ const values: DeepPartial<typeof import('./default').default> = {
...
     typeIn: 'Írj be egy nevet',
   },
+  feedback: {
+    title: 'Küldjön visszajelzést vagy javasoljon új ötletet',
+    feedbackText1: (link: string) =>
+      `Mi egy publikus ${link}ot használunk a visszajelzések és az új ötletek menedzselésére..`,
+    feedbackText2:
+      'Új visszajelzés hozzáadásához egyszerűen küldjön egy e-mailt az alábbi címre, amelyben leírja javaslatát. </br> (email tárgya - a visszajelzés tárgya; email tartalma - visszajelzés leírása)',
+    feedbackText3: (link: string) =>
+      `A nyilvános ${link}-on bármikor ellenőrizheti a meglévő visszajelzéseket és állapotokat.`,
```
Comment (taki9) [inline apps/sensenet/src/localization/hungarian.ts:194]: és azok állapotait

File: apps/sensenet/src/localization/hungarian.ts [hunk 5]
```diff
@@ -180,6 +184,17 @@ const values: DeepPartial<typeof import('./default').default> = {
     name: 'Név',
     typeIn: 'Írj be egy nevet',
   },
+  feedback: {
+    title: 'Küldjön visszajelzést vagy javasoljon új ötletet',
+    feedbackText1: (link: string) =>
+      `Mi egy publikus ${link}ot használunk a visszajelzések és az új ötletek menedzselésére..`,
```
Comment (taki9) [inline apps/sensenet/src/localization/hungarian.ts:190]: -ot
double dot

**Review by taki9 — CHANGES_REQUESTED** [review]

File: apps/sensenet/src/components/appbar/desktop-nav-menu.tsx (hunk 1 above)
Comment (pusztaienike) [reply apps/sensenet/src/components/appbar/desktop-nav-menu.tsx:113]: https://hu.reactjs.org/docs/hooks-reference.html#useref
useRef returns a mutable ref object whose .current property is initialized to the passed argument (initialValue). The returned object will persist for the full lifetime of the component.

File: apps/sensenet/src/components/appbar/desktop-nav-menu.tsx (hunk 3 above)
Comment (pusztaienike) [reply apps/sensenet/src/components/appbar/desktop-nav-menu.tsx:131]: https://material-ui.com/components/popper/#SimplePopper.tsx

File: apps/sensenet/src/localization/hungarian.ts [hunk 6]
```diff
@@ -180,6 +184,17 @@ const values: DeepPartial<typeof import('./default').default> = {
     name: 'Név',
     typeIn: 'Írj be egy nevet',
   },
+  feedback: {
+    title: 'Küldjön visszajelzést vagy javasoljon új ötletet',
```
Comment (molnarm84) [inline apps/sensenet/src/localization/hungarian.ts:188]:
```suggestion
    title: 'Küldj nekünk visszajelzést vagy új ötletet',
```

File: apps/sensenet/src/localization/hungarian.ts [hunk 7]
```diff
@@ -180,6 +184,17 @@ const values: DeepPartial<typeof import('./default').default> = {
...
+    feedbackText1: (link: string) =>
+      `Mi egy publikus ${link}-ot használunk a visszajelzések és az új ötletek menedzselésére.`,
```
Comment (molnarm84) [inline apps/sensenet/src/localization/hungarian.ts:190]:
```suggestion
      `A visszajelzések és ötletek menedzselésére egy publikus ${link}-ot használunk.`,
```

File: apps/sensenet/src/localization/hungarian.ts [hunk 8]
```diff
@@ -180,6 +184,17 @@ const values: DeepPartial<typeof import('./default').default> = {
...
+    feedbackText2:
+      'Új visszajelzés hozzáadásához egyszerűen küldjön egy e-mailt az alábbi címre, amelyben leírja javaslatát. </br> (email tárgya - a visszajelzés tárgya; email tartalma - visszajelzés leírása)',
```
Comment (molnarm84) [inline apps/sensenet/src/localization/hungarian.ts:192]:
```suggestion
      'Új visszajelzés hozzáadásához küldj egy e-mailt az alábbi címre, amelyben leírod a javaslatod. </br> (email tárgya - a visszajelzés tárgya; email tartalma - visszajelzés leírása)',
```

File: apps/sensenet/src/localization/hungarian.ts [hunk 9]
```diff
@@ -180,6 +184,17 @@ const values: DeepPartial<typeof import('./default').default> = {
...
+    feedbackText3: (link: string) =>
+      `A nyilvános ${link}-on bármikor ellenőrizheti a meglévő visszajelzéseket és azok állapotait.`,
```
Comment (molnarm84) [inline apps/sensenet/src/localization/hungarian.ts:194]:
```suggestion
      `A meglévő visszajelzéseket és azok állapotát bármikor ellenőrizheted a nyilvános ${link}-on.`,
```

File: apps/sensenet/src/localization/hungarian.ts [hunk 10]
```diff
@@ -180,6 +184,17 @@ const values: DeepPartial<typeof import('./default').default> = {
...
+    feedbackText4: 'Köszönjük a visszajelzését',
```
Comment (molnarm84) [inline apps/sensenet/src/localization/hungarian.ts:195]:
```suggestion
    feedbackText4: 'Köszönjük a visszajelzést!',
```

**Review by molnarm84 — COMMENTED** [review]
Igyekeztem picit "magyarosítani" a szövegen

**Comment by pusztaienike** [comment]
> Igyekeztem picit "magyarosítani" a szövegen

Köszi 👍

**Review by taki9 — APPROVED** [review]

**Review by CsabaPeter — APPROVED** [review]

**Review by molnarm84 — APPROVED** [review]

**Pull request merged**